corehq/pillows/mappings/user_mapping.py

```diff
@@ -3,7 +3,7 @@
 
 from pillowtop.es_utils import ElasticsearchIndexInfo
 
-USER_INDEX = es_index("hqusers_2017-03-22")
+USER_INDEX = es_index("hqusers_2017-04-24")
```

Comment:
Yeah ok that seems better. I'll revert that and run the reindex manually on production, and softlayer. And coordinate w/ scale team on NIC